Dockerfile.server

```diff
@@ -20,4 +20,4 @@ RUN go install \
     ${NOTARYPKG}/cmd/notary-server
 
 ENTRYPOINT [ "notary-server" ]
-CMD [ "-config", "cmd/notary-server/config.json" ]
+CMD [ "-config", "fixtures/server-config.json" ]
```

Comment:
Are we able to make this the same as the Dockerfile.signer? `"-config=fixtures/server-config.json"` (I actually prefer this syntax to the list with 2 strings, it makes it obvious that `-config` isn't just a flag, it accepts a parameter)